app/components/pipeline/Header/index.js

```diff
@@ -189,7 +174,7 @@ class Header extends React.Component {
     if (this.props.pipeline.description) {
       return <Emojify text={this.props.pipeline.description} />;
     } else {
-      return this.repositoryUrl();
+      return this.props.pipeline.repository.url;
```

Comment:
Huh, that’s weird, it didn’t change anything for me locally, always just output whatever git url was put in. Does that only work if you set up webhooks?